src/utils/webrtc/simplewebrtc/peer.js

```diff
@@ -155,12 +158,260 @@ function preferH264VideoCodecIfAvailable(sessionDescription) {
 	return sessionDescription
 }
 
+// Helper method to munge an SDP to enable simulcasting (Chrome only)
+// Taken from janus.js (MIT license).
+/* eslint-disable */
+function mungeSdpForSimulcasting(sdp) {
+	// Let's munge the SDP to add the attributes for enabling simulcasting
+	// (based on https://gist.github.com/ggarber/a19b4c33510028b9c657)
+	var lines = sdp.split("\r\n");
```

Comment:
We have this lib included:
https://www.npmjs.com/package/sdp-transform

Should make your life easier?

Reply:
Maybe also https://www.npmjs.com/package/@jitsi/sdp-simulcast could be used. For the initial proof-of-concept I was just copying the existing code from Janus.